### DISCUSSION TITLE
automation - many different image themes - different content - queue

### DISCUSSION THREAD

**Discussion by AbcITAndrzej** [lllyasviel/Fooocus#3423] [Q&A] [answered]
Is there a way to specify generation of e.g. multiple image themes e.g.
cat (10 images)
dog (10 images)
car (10 images)

etc

After adding XXX different themes, they will be generated in order?

**Comment by mashb1t** [comment, marked as answer]
What are you referring to when you say "image themes", do you mean styles?
You can use wildcards and processing in order or the array syntax to change content dynamically using the same prompt (examples see readme).

**Comment by AbcITAndrzej** [comment]
I mean the fact that I make a description of a photo and it generates, for example,

Draw a cat.

I click generate.

I have a cat or more (depending on how many images I have selected by default).

I have to wait before generating another image.

I don't want to wait until the system finishes generating a given image, I just want to write, for example: "generate images with the following themes":
dog, cat, elephant, moon, sun, etc.

I upload a list of XXX topics and it generates each topic separately.

Can I do this in some way?

**Reply by mashb1t** [reply]
> You can use wildcards and processing in order or the array syntax to change content dynamically using the same prompt (examples see readme).

as mentioned you can find more information on this in the readme
